lib/membrane/rtp/session_bin.ex

```diff
@@ -695,10 +695,16 @@ defmodule Membrane.RTP.SessionBin do
         struct(Membrane.SRTP.Decryptor, %{policies: state.srtp_policies})
       )
 
-    links_generator = fn twcc? ->
+    links_generator = fn twcc?, ctx ->
+      ssrc_router_pad_options = [
+        encoding: :rtx,
```

Comment:
Is that needed? It looks like SSRC router doesn't make any use of `encoding`...

Reply:
It does. Only to emit `new_inbound_track_event` but it does